src/es6/SirenAction.js

```diff
@@ -3,6 +3,17 @@ import 'd2l-polymer-siren-behaviors/store/entity-store.js';
 import 'd2l-polymer-siren-behaviors/store/action-queue.js';
 import SirenParse from 'siren-parser';
 
+/**
+ * @typedef {import('siren-parser').Action} Action
+ * @typedef {import('siren-parser').Field} Field
+ * @typedef {import('siren-parser').Entity} ParsedEntity
+ * @typedef {{ name: string; value: string }} FieldOverride
```

Comment:
Intuitively it seems like this shouldn't be necessary i.e. should be able to just use `Field` - but I assume something is missing somewhere that prevents that?

Reply:
Field also includes `hasClass`

Reply:
shoot, right - the exports of `siren-parser` really should have included both `ActualSirenEntity` and `SirenParserEntity`, ah well (this is directed at myself)